apps/pg_subscriptor/test/handler.exs

```diff
@@ -0,0 +1,13 @@
+defmodule PgHandlerTest do
+  use ExUnit.Case
+  doctest PgHandler
+
+  test "insert message" do
+    # TODO: binary representation of columns
+    message = <<"I", 1::32, "N", "%{id: 1, name: Juraj}"::binary>>
```

Comment:
Now I see why you added tuple data directly to the insert message. Yeah leave it as is in this PR.